samples/spring-security-basic-auth/src/main/java/sample/spring/xsuaa/TestController.java

```diff
@@ -32,4 +32,9 @@
 		result.put("scopes", String.valueOf(token.getScopes()));
 		return result;
 	}
+
+	@GetMapping("/health")
```

Comment:
was this required by K8S?

Reply:
nope, but I really like to have a health endpoint, it helps to analyze issue in case app doesn't work properly, mainly for new feature development and testing

Reply:
all right :-)